rubygem-katello/rubygem-katello.spec

```diff
@@ -13,7 +13,7 @@ Name:    %{?scl_prefix}rubygem-%{gem_name}
 Summary: Katello
 
 Version: 2.4.0
-Release: 1.nightly%{dist}
+Release: 1.rc1%{dist}
```

Comment:
Again not trying to be picky, just understanding, so 6.nightly could have
been bumped to 6.rc1 ?
On Oct 25, 2015 9:17 AM, "Justin Sherrill" <notifications@github.com> wrote:

> In rubygem-katello/rubygem-katello.spec
> <https://github.com/Katello/katello-packaging/pull/88#discussion_r42944705>
> :
>
> > @@ -13,7 +13,7 @@ Name:    %{?scl_prefix}rubygem-%{gem_name}
> >  Summary: Katello
> >
> >  Version: 2.4.0
> > -Release: 1.nightly%{dist}
> > +Release: 1.rc1%{dist}
>
> no IMHO, as 1.nightly is upgradable to 1.rc1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/katello-packaging/pull/88/files#r42944705>.
>